Content.Server/Administration/Commands/PlayTimeCommands.cs

```diff
@@ -123,6 +125,83 @@ public CompletionResult GetCompletion(IConsoleShell shell, string[] args)
     }
 }
 
+[AdminCommand(AdminFlags.Moderator)]
+public sealed class PlayTimeAddDepartmentCommand : IConsoleCommand
```

Comment:
This needs to use Toolshed instead. `IConsoleCommand` will likely be removed in the future.

Reply:
we have a (albeit very incomplete) toolshed doc in the wiki https://docs.spacestation14.com/en/robust-toolbox/toolshed.html which explains the basics, but yeah you will have to look at examples from the codebase, feel free to ask questions if you get stuck somewhere in the process

Reply:
Ok, I'll try

Reply:
> This needs to use Toolshed instead. `IConsoleCommand` will likely be removed in the future.

This is absolutely wrong. We have no plans to remove `IConsoleCommand`.